src/Testcontainers.EventStore/EventStoreBuilder.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Testcontainers.EventStore
+{
+  public class EventStoreBuilder : ContainerBuilder<EventStoreBuilder, EventStoreContainer, EventStoreConfiguration>
+  {
+    public const string EventStoreImage = "eventstore/eventstore:21.2.0-buster-slim";
+
+    public const ushort EventStorePort = 2113;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+    /// </summary>
+    public EventStoreBuilder()
+      : this(new EventStoreConfiguration())
+    {
+      DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+    /// </summary>
+    /// <param name="dockerResourceConfiguration">The Docker resource configuration.</param>
+    private EventStoreBuilder(EventStoreConfiguration dockerResourceConfiguration)
+      : base(dockerResourceConfiguration)
+    {
+      DockerResourceConfiguration = dockerResourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override EventStoreConfiguration DockerResourceConfiguration { get; }
+
+
+    /// <inheritdoc />
+    public override EventStoreContainer Build()
+    {
+      return new EventStoreContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+    }
+
+    /// <inheritdoc />
+    protected override EventStoreBuilder Init()
+    {
+      return base.Init()
+        .WithImage(EventStoreImage)
+        .WithPortBinding(EventStorePort, true)
+        .WithEnvironment("EVENTSTORE_CLUSTER_SIZE", "1")
+        .WithEnvironment("EVENTSTORE_RUN_PROJECTIONS", "All")
+        .WithEnvironment("EVENTSTORE_START_STANDARD_PROJECTIONS", "true")
+        .WithEnvironment("EVENTSTORE_EXT_TCP_PORT", "1113")
+        .WithEnvironment("EVENTSTORE_EXT_HTTP_PORT", "2113")
+        .WithEnvironment("EVENTSTORE_INSECURE", "true")
+        .WithEnvironment("EVENTSTORE_ENABLE_EXTERNAL_TCP", "true")
+        .WithEnvironment("EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP", "true")
+        .WithWaitStrategy(Wait.ForUnixContainer().UntilPortIsAvailable(EventStorePort));
```

Comment:
Did you check if it is enough to wait until the port is available? From my experience usualy ports are available before the service is ready.

Reply:
I think we must wait until container is ready and after that set the env variable for that.
```csharp
base.Init()
        .WithImage(EventStoreImage)
        .WithPortBinding(EventStorePort, true)
        .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
        .WithEnvironment("EVENTSTORE_CLUSTER_SIZE", "1")
        .WithEnvironment("EVENTSTORE_RUN_PROJECTIONS", "All")
        .WithEnvironment("EVENTSTORE_START_STANDARD_PROJECTIONS", "true")
        .WithEnvironment("EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP", "true")
        .WithEnvironment("EVENTSTORE_INSECURE", "true");
```
I don't know use `UntilContainerIsHealthy` is better or `UntilPortIsAvailable`!? But without `WithWaitStrategy` we can't handle `env` and fails container.

Reply:
`UntilContainerIsHealthy` looks good. I double-checked it. Maybe we need to increase the failing streak, but for now it looks good.